court/migrations/0001_initial.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2 on 2023-04-12 17:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Court',
+            fields=[
```

Comment:
Why only the `id` field is mentioned here? Where are the rest of the fields?

Reply:
Fixed. Thank you for the reference.